tldw_Server_API/app/core/config.py

```
@@ -494,6 +504,17 @@ def _to_bool(value: Optional[str], default: bool = False) -> bool:
             return default
         return str(value).strip().lower() in {"1", "true", "yes", "on"}
 
+    def _safe_int(value: object, default: int) -> int:
+        try:
+            if value is None:
+                return default
+            s = str(value).strip()
+            if not s:
+                return default
+            return int(s)
+        except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)